.circleci/config.yml

```diff
@@ -960,13 +1107,108 @@ jobs:
           path: docs/state-channels
       - *fail_notification
 
+  # Windows jobs
+  win_build: &windows_build
+    <<: *windows_container
+    steps:
+      - windows_install_deps:
+          otp_install_path: << parameters.otp_install_path >>
+          otp_version: << parameters.otp_version >>
+          erts_version: << parameters.erts_version >>
+          msys_install_path: << parameters.msys_install_path >>
+      - run:
+          name: Build
+          command: .circleci\windows\build.bat
+      - store_artifacts:
+          path: *windows_artifacts
+          destination: /tmp/packages-<< parameters.otp_version >>
+      - persist_to_workspace:
+          root: *windows_artifacts
+          paths:
+            - "*.zip"
+            - "*.exe"
+
+  win_test: &windows_test
+    <<: *windows_container
+    steps:
+      - windows_install_deps:
+          otp_install_path: << parameters.otp_install_path >>
+          otp_version: << parameters.otp_version >>
+          erts_version: << parameters.erts_version >>
+          msys_install_path: << parameters.msys_install_path >>
+      - attach_workspace:
+          at: *windows_artifacts
+      - run:
+          name: Smoke test
+          command: .circleci\windows\test.bat
+      - store_artifacts:
+          path: *windows_artifacts
+          destination: /tmp/packages-<< parameters.otp_version >>
+
+  win_build_20_1: *windows_build
```

Comment:
We need to create something like aliases for the jobs, because config requires every job to have unique key. This is kind of hackish